tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "ES2018",
```

Comment:
turns out storybook 6 does not support conditional chaining. `npm run storybook:build` gave me issues.  i updated to target `ES2018` to get things working.  https://github.com/storybookjs/storybook/issues/11515